### PR TITLE
added fix for claude 3, included messageAPI

### PR DESCRIPTION
**Pull request by patraxo**
*Issue #, if available:*

*Description of changes:*

Included messageAPI and fixed the issues of incompatibility with claude 3 models

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: pca-ui/src/genai/index.py [hunk 1]
```diff
@@ -90,7 +103,7 @@ def get_bedrock_generate_text(modelId, response):
         generated_text = response_body.get("results")[0].get("outputText")
     else:
         raise Exception("Unsupported provider: ", provider)
-    generated_text = generated_text.replace('```','')
+    #generated_text = generated_text.replace('```','')
```
Comment (kishd) [inline pca-ui/src/genai/index.py:106]: Any reason for commenting this?

**Comment by chrislott** [comment]
Related to #259

File: pca-ui/src/genai/index.py (hunk 1 above)
Comment (patraxo) [reply pca-ui/src/genai/index.py:106]: Let me uncomment it

Comment (patraxo) [reply pca-ui/src/genai/index.py]: changes made, please check now

**Pull request merged**